requirements.txt

```diff
@@ -1,9 +1,9 @@
--e git://github.com/tejado/pgoapi.git@1f25e907f3e5f1b603330e71041e1ad7bee7580f#egg=pgoapi
+-e git+https://github.com/tejado/pgoapi.git#egg=pgoapi
```

Comment:
We need fixed on specific version of pgoapi, before upgrade, test need to be done.